plugins/sy-personio-jobs-source-plugin/gatsby-node.js

```diff
@@ -0,0 +1,32 @@
+const { fetch } = require('./personio-jobs-api');
+const JOB_TYPE = 'SyPersonioJob';
+
+exports.onPreInit = () => console.log('Loaded sy-personio-jobs-source-plugin');
+
+exports.sourceNodes = async ({
+  actions,
+  createContentDigest,
+  createNodeId,
+}) => {
+  const { createNode } = actions;
+
+  const positionsDE = await fetch('de');
+  const positionsEN = await fetch('en');
+
+  const positions = positionsDE.concat(positionsEN);
+
+  positions.forEach((position) => {
+    createNode({
+      ...position,
+      jobId: position.id,
+      id: createNodeId(`${JOB_TYPE}-${position.lang}-${position.id}`),
+      parent: null,
+      children: [],
+      internal: {
+        type: JOB_TYPE,
+        content: JSON.stringify(position),
+        contentDigest: createContentDigest(position),
+      },
+    });
+  });
+};
```

Comment:
that's the only magic need to make our jobs available through graphl. Fetch all datasets and then push them into the underlying redux store with `createNode` and gatsby does the rest of inferring the graphql schema. This is awesome everytime I think about it.